provisioning/tasks/sshd.yml

```diff
@@ -5,3 +5,18 @@
     regexp: "^AcceptEnv LANG"
     state: absent
     validate: "/usr/sbin/sshd -T -f %s"
+
+# Copy over known_hosts.
```

Comment:
This comment can be left out; it's obvious enough from the two tasks what's going on :)